Corona-Warn-App/src/test/java/de/rki/coronawarnapp/environment/BuildConfigWrapTest.kt

```diff
@@ -1,13 +1,24 @@
 package de.rki.coronawarnapp.environment
 
-import io.kotest.matchers.collections.shouldBeIn
+import de.rki.coronawarnapp.BuildConfig
+import io.kotest.matchers.shouldBe
 import org.junit.jupiter.api.Test
 import testhelpers.BaseTest
 
 class BuildConfigWrapTest : BaseTest() {
 
     @Test
-    fun `default environment type should be DEV`() {
-        BuildConfigWrap.ENVIRONMENT_TYPE_DEFAULT shouldBeIn listOf("DEV", "INT", "WRU-XD", "PROD")
+    fun `default environment type `() {
+        if (BuildConfig.FLAVOR == "deviceForTesters" && BuildConfig.BUILD_TYPE == "debug") {
+            BuildConfigWrap.ENVIRONMENT_TYPE_DEFAULT shouldBe "INT"
+        } else if (BuildConfig.FLAVOR == "deviceForTesters" && BuildConfig.BUILD_TYPE == "release") {
+            BuildConfigWrap.ENVIRONMENT_TYPE_DEFAULT shouldBe "WRU"
+        } else if (BuildConfig.FLAVOR == "device" && BuildConfig.BUILD_TYPE == "debug") {
+            BuildConfigWrap.ENVIRONMENT_TYPE_DEFAULT shouldBe "INT"
+        } else if (BuildConfig.FLAVOR == "device" && BuildConfig.BUILD_TYPE == "release") {
+            BuildConfigWrap.ENVIRONMENT_TYPE_DEFAULT shouldBe "PROD"
+        } else {
+            throw IllegalStateException("Unknown flavor/build combination.")
+        }
```

Comment:
Looks like this cab be `when`  ?

Reply:
yes, not having too big of an impact here. Let's fix it when this is touched again :grin: